config/plop/templates/page/Page.fragment.graphql.hbs

```diff
@@ -0,0 +1,29 @@
+fragment {{ pascalCase name }}Route on {{ pascalCase name }}Record {
+  __typename
+  id
+  title
+  {{#each (routeParams route)}}
+  {{#if (eq (trailingParam ../route) this)}}
+  slug
```

Comment:
theoretically I could've reassigned slug to the trailingParam, but I felt that to be overcomplicating it, and I must say I think it makes more sense to promote the use of `[slug]` as final param